packages/nodes-base/nodes/Xero/Xero.node.ts

```diff
@@ -538,41 +538,41 @@ export class Xero implements INodeType {
 						body.xeroNetworkKey = additionalFields.xeroNetworkKey as string;
 					}
 
-					// if (phonesUi) {
-					// 	const phoneValues = phonesUi?.phonesValues as IDataObject[];
-					// 	if (phoneValues) {
-					// 		const phones: IPhone[] = [];
-					// 		for (const phoneValue of phoneValues) {
-					// 			const phone: IPhone = {};
-					// 			phone.Type = phoneValue.type as string;
-					// 			phone.PhoneNumber = phoneValue.PhoneNumber as string;
-					// 			phone.PhoneAreaCode = phoneValue.phoneAreaCode as string;
-					// 			phone.PhoneCountryCode = phoneValue.phoneCountryCode as string;
-					// 			phones.push(phone);
-					// 		}
-					// 		body.Phones = phones;
-					// 	}
-					// }
-
-					// if (addressesUi) {
-					// 	const addressValues = addressesUi?.addressesValues as IDataObject[];
-					// 	if (addressValues) {
-					// 		const addresses: IAddress[] = [];
-					// 		for (const addressValue of addressValues) {
-					// 			const address: IAddress = {};
-					// 			address.Type = addressValue.type as string;
-					// 			address.AddressLine1 = addressValue.line1 as string;
-					// 			address.AddressLine2 = addressValue.line2 as string;
-					// 			address.City = addressValue.city as string;
-					// 			address.Region = addressValue.region as string;
-					// 			address.PostalCode = addressValue.postalCode as string;
-					// 			address.Country = addressValue.country as string;
-					// 			address.AttentionTo = addressValue.attentionTo as string;
-					// 			addresses.push(address);
-					// 		}
-					// 		body.Addresses = addresses;
-					// 	}
-					// }
+					if (phonesUi) {
```

Comment:
maybe create a function here as to not have the same code in multiple places

Reply:
@Jhalter5Stones I would agree with you but do we want to break constancy with how they are handling these functionalities for everything else?